src/pages/nfts/collections/[collectionAddress].tsx

```diff
@@ -23,15 +24,13 @@ export const getStaticProps: GetStaticProps = async ({ params }) => {
   }
 
   try {
-    await store.dispatch(fetchCollection(collectionAddress))
-
-    const state = store.getState()
+    const collectionsData = await getCollections()
+    const collectionData = await getCollection(collectionAddress)
 
     return {
       props: {
-        // Note: only include the data needed for the page, otherwise it will override the user preset data
-        initialReduxState: {
-          nftMarket: state.nftMarket,
+        fallback: {
+          [unstable_serialize(['nftMarket', 'collections'])]: { ...collectionsData, ...collectionData },
```

Comment:
IMO better separate collections `['nftMarket', 'collections']` and collection `['nftMarket', 'collections', collectionAddress]`

Reply:
Done